### ISSUE TITLE
 Class 'ar727' not found

### ISSUE THREAD

**Issue by flrfnc** [oommgg/Soyal#1]
Hi,

i've a problem:

C:\Soyal-1.0.1>php -a
Interactive shell

php > require __DIR__ . '/vendor/autoload.php';
php > use Oommgg\Soyal\Ar727;
php > $soyal = new ar727('172.17.19.133', 1621);

Warning: Uncaught Error: Class 'ar727' not found in php shell code:1
Stack trace:
#0 {main}
  thrown in php shell code on line 1

can you help me?

Thanks
Francesco

**Comment by oommgg** [comment]
Hi there,



You can try put the code in a single PHP file, e.g. test.php

```php
require __DIR__ . '/vendor/autoload.php';

use Oommgg\Soyal\Ar727;

$soyal = new Ar727('192.168.1.66', 1621);

$result = $soyal->getTime();

var_dump($result);
```

then run the code via, 
```
C:\>php test.php
```

**Comment by flrfnc** [comment]
Hi,
thanks for your time.

new problem

my soyal parameter are:
soyal ar727h ip=172.17.19.133 port= 1621 nodeid=1 mastercode=123456

the file:

opadmin@soyalphp:~$ more test.php


 require __DIR__ . '/vendor/autoload.php';
 use Oommgg\Soyal\Ar727;
 $soyal = new Ar727('172.17.19.133', 1621);
 $result = $soyal->getTime();
 var_dump($result);


the result:

opadmin@soyalphp:~$ php test.php

PHP Fatal error:  Uncaught Oommgg\Soyal\Exceptions\DeviceErrorException: Node error: can not get node data. in /home/opadmin/vendor/oommgg/soyal/src/Ar727.php:98
Stack trace:
#0 /home/opadmin/vendor/oommgg/soyal/src/Ar727.php(258): Oommgg\Soyal\Ar727->receive()
#1 /home/opadmin/test.php(5): Oommgg\Soyal\Ar727->getTime()
#2 {main}
  thrown in /home/opadmin/vendor/oommgg/soyal/src/Ar727.php on line 98
opadmin@soyalphp:~$

why cannot get data?

Thanks
Francesco

**Comment by oommgg** [comment]
Hi @flrfnc ,

please check, is your `AR727`  same submask with your development env.?, 
Therefore,  Can you login the  `AR727` via the device's web platefrom?

try PING the device for the response,  
```shell
 C:\PING 172.17.19.133 
```
And, please make sure the `PORT` is free for connect. The PORT number `1621` is not the default PORT number, please notice this... you have to check the settings from the `AR727`'.

if you are running the code under the `DOCKER`, please make sure you have open the `PORT` for it.

**Comment by flrfnc** [comment]
Hi Ben,

the device responds to the ping, the port is 1621 and it's open.
I set the nodeid to 1 for convenience as I saw on the code that this is your default setting.

The device is not on the same subnet but connection takes place.
However no data is received and the buffer is empty:
.....
if (empty($buffer)) {
            throw new DeviceErrorException('Node error: can not get node data.');
        }
....
where am i wrong?

Thanks
Francesco
